apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html

```diff
@@ -1610,39 +1962,81 @@
                     <div class="table-cell">Total Amount</div>
                     <div class="table-cell figure-cell">
                         <span
-                            *ngIf="isMulticurrencyAccount && isMultiCurrencySupported">{{purchaseOrder.accountDetails.currencySymbol}}</span>{{purchaseOrder.voucherDetails.subTotal | giddhCurrency}}
+                            *ngIf="isMulticurrencyAccount && isMultiCurrencySupported">{{purchaseOrder.accountDetails.currencySymbol}}
+                        </span>{{purchaseOrder.voucherDetails.subTotal | giddhCurrency}} shal
```

Comment:
if amount-field component is used already for this, this should be removed/commented

Reply:
commented